src/coreclr/pal/inc/pal.h

```diff
@@ -628,23 +628,6 @@ PAL_PerfJitDump_Finish();
 #define IDYES                   6
```

Comment:
Are these message box specific constants still needed on Unix?

Reply:
Yes, sadly. I tried to remove them. If I can follow some of your other suggestions, I may be able to remove them.

Reply:
I'm investigating this now.

Reply:
I've removed most of them. They are used in some debugger scenarios so I am going to leave the rest for now.